client/src/components/common/UserAvatar/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactElement } from 'react';
+import { Image } from 'semantic-ui-react';
+
+import styles from './styles.module.scss';
+import { getInitials } from 'helpers/getInitials.helper';
+
+interface Props {
+	user: WebApi.Entities.UserProfile;
+	small?: boolean;
+}
+
+function UserAvatar({ user, small }: Props): ReactElement {
+	return (
+		<>
+			{user.avatar ? (
+				<Image src={user.avatar} className={[small && styles.small, styles.avatarImage].join(' ')} circular />
```

Comment:
Use [classnames](https://www.npmjs.com/package/classnames) package for such cases.
```suggestion
				<Image src={user.avatar} className={classnames(styles.avatarImage, { [styles.small]: small })} circular />
```

Reply:
Сompleted